cognite/powerops/custom_modules/power_model_v1/data_models/containers/Case.container.yaml

```
@@ -10,19 +10,51 @@ properties:
     autoIncrement: false
     name: scenario
     description: The scenario to use when running this case
+  case:
+    type:
+      list: false
+      type: file
+    nullable: true
+    autoIncrement: false
+    name: case
+    description: The resulting case file to send to shop after preprosessor has performed mappings on model template
+  reservoirMapping:
+    type:
+      list: false # Only one per case?
+      type: file
+    nullable: true
+    autoIncrement: false
+    name: reservoirMapping
+    description: Cut file reservoir mapping
+  moduleSeries:
```

```suggestion
  cutOrderFile:
```
Modulserie is a word that HEV uses (maybe it comes from ProdRisk), but for SHOP, this is called CUTORDER (I think).
I don't know if it makes sense to have more than one of these, but I haven't really looked into it.